x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_list/components/alerts_list.tsx

```diff
@@ -353,7 +421,110 @@ export const AlertsList: React.FunctionComponent = () => {
           </EuiFlexGroup>
         </EuiFlexItem>
       </EuiFlexGroup>
-
+      <EuiSpacer size="m" />
+      {!dissmissAlertErrors && alertsStatusesTotal[AlertExecutionStatusValues[2]] > 0 ? (
+        <EuiFlexGroup>
+          <EuiFlexItem>
+            <EuiCallOut
+              color="danger"
+              size="s"
+              title={
+                <FormattedMessage
+                  id="xpack.triggersActionsUI.sections.alertsList.attentionBannerTitle"
+                  defaultMessage="There is {totalStausesError} {totalStausesError, plural, one {{singleTitle}} other {# {multipleTitle}}} with an error."
+                  values={{
+                    totalStausesError: alertsStatusesTotal[AlertExecutionStatusValues[2]],
+                    singleTitle: 'alert',
+                    multipleTitle: 'alerts',
+                  }}
+                />
+              }
+              iconType="alert"
+            >
+              <EuiButton
+                type="primary"
+                size="s"
+                color="danger"
+                onClick={() => setAlertStatusesFilter([AlertExecutionStatusValues[2]])}
+              >
+                <FormattedMessage
+                  id="xpack.triggersActionsUI.sections.alertsList.viewBunnerButtonLabel"
+                  defaultMessage="View"
+                />
+              </EuiButton>
+              <EuiButtonEmpty color="danger" onClick={() => setDissmissAlertErrors(true)}>
+                <FormattedMessage
+                  id="xpack.triggersActionsUI.sections.alertsList.dismissBunnerButtonLabel"
+                  defaultMessage="Dismiss"
+                />
+              </EuiButtonEmpty>
+            </EuiCallOut>
+          </EuiFlexItem>
+        </EuiFlexGroup>
+      ) : null}
+      <EuiSpacer size="m" />
+      <EuiFlexGroup>
+        <EuiFlexItem grow={false}>
+          <EuiText size="s" color="subdued">
+            <FormattedMessage
+              id="xpack.triggersActionsUI.sections.alertsList.totalItemsCountDescription"
+              defaultMessage="Showing: {pageSize} of {totalItemCount} alerts."
```

Comment:
Should this use a variable other than pageSize? I am seeing `Showing: 10 of 1 alerts` when I only have one alert.
<img width="1169" alt="Screen Shot 2020-10-02 at 9 14 13 AM" src="https://user-images.githubusercontent.com/13104637/94929485-286f0480-0493-11eb-9cd0-01ce1f162df9.png">

Reply:
Make sense! Thanks for catching, I really forgot about this part.